source/manual/alerts/rabbitmq-no-consumers-listening.html.md

```diff
@@ -4,75 +4,94 @@ title: 'RabbitMQ: No consumers listening to queue'
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2019-12-05
+last_reviewed_on: 2020-09-07
 review_in: 6 months
 ---
 
-[Read more about how we use RabbitMQ][rabbitmq]
+## Check that there is at least one non-idle consumer for rabbitmq queue <queue>
 
-We run a check that there is at least one non-idle consumer for a named
-RabbitMQ queue. (For example, for the `email_alert_service` queue.) The queue
-name should indicate the app responsible for consuming the queue.
+See [here][check_rabbitmq_plugin] for the plugin that implements the alert.
 
-The Icinga check is performed by connecting to RabbitMQ's admin API, so the
-information given is from Rabbit's point of view.
+For information about how we use RabbitMQ, see [here][rabbitmq_doc].
 
-Since we send heartbeats every minute (from each instance of the Publishing
-API) at least one of the consumers for each queue should be active every
-minute. To be conservative, the check checks that at least one consumer of the
-queue is running and has been active within the last 5 minutes.
+### `No consumers listening to queue`
 
-If the check succeeds, it will return the most recent time at which activity
-happened on the queue.
+This check reports a critical error when **no** consumers are listening to the
+queue, meaning messages entering the queue will never be processed. Icinga
+connects to RabbitMQ's admin API to check that at least one consumer is running
+for a given RabbitMQ message queue.
 
-If the check fails due to no consumers having been active recently, it will
-report a critical failure, giving the most recent idle time, and the idle
-times of each consumer connected to RabbitMQ:
+### `No activity for X seconds: idle times are [X, Y, Z]`
 
-```
-CRITICAL: No activity for X seconds: idle times are [X, Y, Z]
-```
+This checks whether the RabbitMQ consumers for a queue have been active in the
+last 5 minutes. Consumers in an `idle` state are listening to the queue but are
+unable to process the messages on it. Icinga connects to RabbitMQ's admin API
+to check on the activity of the consumers of a given RabbitMQ message queue.
 
-If the check fails due to there being no consumers connected to the queue, it
-will report a critical failure:
+Publishing API [sends a heartbeat message][heartbeat_messages] every minute to
+the following queues - `email-alert-service`, `cache_clearing_service-high` and
+`content_data_api`. These queues receive the heartbeat due to being classed as
```

Comment:
Minor: this is a bit convoluted - my original suggestion wasn't great. We could just say "... because they listen for `*.major` messages".

Reply:
I've just got rid of the sentence as I don't think it adds much value as the sentence after it describes how it's configured.